DOCKER.md

```diff
@@ -0,0 +1,44 @@
+# Docker Setup
+========================
+
+This document describes how to buidl and run a minimal InfluxDB container under Docker.   It has currently only be tested for local development and assumes that you have a working docker environment already.
```

Comment:
Change 

`It has currently only be tested for local development and assumes that you have a working docker environment already.` 

to

`Currently, it has only been tested for local development and assumes that you have a working docker environment.`